tests/acceptance/find-lazy-module.spec.ts

```diff
@@ -0,0 +1,48 @@
+import * as mockFs from 'mock-fs';
+import {stripIndents} from 'common-tags';
+import {expect} from 'chai';
+
+import {findLazyModules} from '../../addon/ng2/models/find-lazy-modules';
+
+
+describe('find-lazy-module', () => {
+  beforeEach(() => {
+    mockFs({
+      'project-root': {
+        'fileA.ts': stripIndents`
+          const r1 = {
+            "loadChildren": "moduleA"
+          };
+          const r2 = {
+            loadChildren: "moduleB"
+          };
+          const r3 = {
+            'loadChildren': 'moduleC'
+          };
+          const r4 = {
+            "loadChildren": 'app/+workspace/+settings/settings.module#SettingsModule'
```

Comment:
@hansl - can the value of the `loadChildren` key be any expression (that evaluates to a string) or must it be a string literal?

cc @ericjim